apparelo/apparelo/doctype/dc/dc.py

```diff
@@ -510,8 +512,10 @@ def get_expected_items_in_return(doc, items_to_be_sent=None, use_delivery_qty=Fa
 		if not use_delivery_qty:
 			item_to_be_received['projected_qty'] = item_to_be_received['qty']
 			item_to_be_received['qty'] = 0
-	
-	items_to_be_received = sorted(items_to_be_received, key = lambda i: i['attribute']) 
+	if 'Apparelo Colour' in item_to_be_received and 'Dia' in item_to_be_received:
+		items_to_be_received = sorted(sorted(items_to_be_received, key = lambda i: i['Dia']), key = lambda i: i['Apparelo Colour'])
```

Comment:
Please handle cases where only 'Dia' is there as an attribute.

Reply:
Check this commit https://github.com/aerele/apparelo/pull/182/commits/fad746d2885ad2560b0b0562f331bf9b0246b02c

Reply:
looks good... cool...